data/filesystem/autoyast_filesystem_withouthome.xml.ep

```diff
@@ -6,7 +6,7 @@
     <email/>
     <reg_code>{{SCC_REGCODE}}</reg_code>
     <install_updates config:type="boolean">true</install_updates>
-    % unless ($check_var->('FLAVOR') =~ /-Updates$|-Incidents/) {
+    % unless ($check_var->('FLAVOR', '') =~ /-Updates$|-Incidents/) {
```

Comment:
Just for completeness - the result of `check_var` is a perl boolean, 1 or the empty string, so that will never match the regex, so it should be fixed at some point.
But the change reflects the current behaviour, so good.

Reply:
follow up on  #15064